### PR TITLE
feat: slider

### PR DESCRIPTION
**Pull request by harshitadaswani**
Added slider component
preview-link: https://deploy-preview-16--harshita-crypto-ui.netlify.app/components/slider.html

### CODE REVIEW

File: components/slider.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+.slidecontainer {
+    width: 100%; 
+}
+
+.slider {
+    width: 90%;
+    height: 1.5rem;
+    background-color: var(--lynx-grey-color);
```
Comment (JantuDeb) [inline components/slider.css:14]: This will not work I guess.

**Review by JantuDeb — APPROVED** [review]
Code looks good

File: components/slider.css (hunk 1 above)
Comment (harshitadaswani) [reply components/slider.css:14]: will change

**Review by tushartiwari7 — APPROVED** [review]
Good to merge!

File: components/slider.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+</head>
+<body>
+    <div class="p-s container">
+
+        <nav class="navigation">
+            <div class="nav-left">
+                <a href="../index.html" class="txt-l fw-black p-s m-s">Crypto&nbsp;UI</a>
+            </div>
+            <div class="nav-right">
+                <a href="https://github.com/harshitadaswani" target="_blank" rel="noopener noreferrer" class="txt-l fw-black p-s m-s"><i class="fab fa-github-square icons"></i></i></a>
```
Comment (Priya-Patel31) [inline components/slider.html:19]: remove one of the closing tag of <i> as there are two.

File: components/slider.html [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            </aside>
+
+        <main class="main-container">
+            <h1 class="h1-title">Slider</h1>
+            <p class="border-bottom"> &lt;input> elements of type range let the user specify a numeric value which must be no less than a given value, and no more than another given value. </p>
+            <div class="example-div br-m">
+                <div>
+                    <div class="fw-bold txt-m m-s p-s">Price</div>
+                    <div class="slider-label m-s p-s">
+                        <span>100</span>
```
Comment (Priya-Patel31) [inline components/slider.html:64]: avoid using non-semantic tags i.e span

File: components/slider.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
+.slider-label{
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+}
+
+.slidecontainer {
```
Comment (Priya-Patel31) [inline components/slider.css:7]: Just a suggestion slider- container could be a better name rather than slidecontainer

**Review by Priya-Patel31 — COMMENTED** [review]
The UI and code both looks clean . Just need to make some minor changes suggested by me.

**Pull request merged**